### ISSUE TITLE
Batch load images and auto-queue them for the same workflow.

### ISSUE THREAD

**Issue by ilanhuang** [comfyanonymous/ComfyUI#13006]
### Feature Idea

It would be great to have a batch loading image option in load image node and then when I start queuing the workflow it automatically follows the image batch sequence.

Some times I need to run the same workflow for a big set of images and when I automate it starts X comfy sessions depending on the batch size making it way slower and hardware consuming than just open one session and load one image at a time. 

Does comfy already have a solution for this? 

For example: I have a workflow that removes and in-paint 360 selfie sticks and operators for gaussian splatting pre-training processes, I often have to process 200+ images per set.

**Comment by njcray2** [comment]
FWIW here is what I do when I want to batch process a lot of jpgs through OCR. It might give you some ideas for your needs.

First use the "Folder Search" node to get a full list of all the images. You can use wildcards and search recursively if you need to.
Then use "Index Any" to pull out the nth entry. If you change the "control after generate" to "increment" then every time you click "Run" it will step to the next image in the list. And you can arrange for a click of "run" to schedule up to 100 runs at a time, which will run sequentially. Just remember that the pointer moves on after each click. It is very easy to get confused about which file it has just selected! In the image the file just processed is not the same as the index value. That is usually the case!

<img width="1225" height="639" alt="Image" src="https://github.com/user-attachments/assets/e9db3b6d-6a39-41f4-9aa2-b3bbeec9a930" />

Now you have the filename you can use it to fetch the image you want:

<img width="731" height="387" alt="Image" src="https://github.com/user-attachments/assets/13c9841a-8201-48ac-ab0d-983ca54fc6f3" />

But the big advantage (in my case anyway) is that I can process the original filename to create an output filename which is related to the file we started with:

<img width="1208" height="577" alt="Image" src="https://github.com/user-attachments/assets/29dc25c3-aa2b-46c0-992b-c124aef5bdca" />
